tests/acceptance/features/bootstrap/Sharing.php

```diff
@@ -1962,6 +1962,17 @@ public function userGetsAllSharesSharedByHimUsingTheSharingApi(string $user):voi
 		$this->setResponse($this->getAllshares($user));
 	}
 
+	/**
+	 * @When user :user gets all the shared folders
+	 *
+	 * @param string $user
+	 *
+	 * @return void
+	 */
+	public function userGetsAllTheSharedFolders($user) {
+		$this->setResponse($this->getAllshares($user));
+	}
```

Comment:
let's not use Sharing context for new steps related to shares. we have to use SharingNG (graph endpoint)